content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -388,8 +388,8 @@ to 1 second. Minimum value is 1.
 * `successThreshold`: Minimum consecutive successes for the probe to be
 considered successful after having failed. Defaults to 1. Must be 1 for liveness
 and startup Probes. Minimum value is 1.
-* `failureThreshold`: When a probe fails, Kubernetes will
-try `failureThreshold` times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready.
+* `failureThreshold`: When a probe fails `failureThreshold` times in a row, Kubernetes will give up.
+Give up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready.
 Defaults to 3. Minimum value is 1.
```

Comment:
How about this text:
```suggestion
* `failureThreshold`: After a probe fails `failureThreshold` times in a row, Kubernetes
  considers that the overall check has failed: the container is _not_ ready / healthy /
  live.
  For the case of a startup or liveness probe, if at least `failureThreshold` probes have
  failed, Kubernetes treats the container as unhealthy and triggers a restart for that
  specific container. The kubelet takes the setting of `terminationGracePeriodSeconds`
  for that container into account.
  For a failed readiness probe, the kubelet continues running the container that failed
  checks, and also continues to run more probes; because the check failed, the kubelet
  sets the `Ready` [condition](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions)
  on the Pod to `false`.
* `terminationGracePeriodSeconds`: configure a grace period for the kubelet to wait
  between triggering a shut down of the failed container, and then forcing the
  container runtime to stop that container.
  The default is to inherit the Pod-level value for `terminationGracePeriodSeconds`
  (30 seconds if not specified), and the minimum value is 1.
  See [probe-level `terminationGracePeriodSeconds`](#probe-level-terminationgraceperiodseconds)
  for more detail.
```

Reply:
+1 to sftim's [suggestion](https://github.com/kubernetes/website/pull/37416/files#r1005607421)

Reply:
So, I completely overlooked your comment. It is a  very good suggestion. Committed. Thanks!